### ISSUE TITLE
General comments

### ISSUE THREAD

**Issue by inikitin** [tapbots/calcbot-localization#27]
I've noticed a few English typos during translation:

'Belarussian Ruble' —> 'Belarusian Ruble'
'Russian Rouble' —> 'Russian Ruble'

Another thing is that when I was trying to figure out what 'Renminbi' is, it turned out it's the official name of Chinese currency, and Yuan (the name by which, I assume, most of us know Chinese currency) is just its base unit. I'd suggest to drop 'Renminbi' from the list, since it's basically just a duplicate.

Also, on the list there are several currencies that no longer exist, like most European ones (French Franc, Italian Lira, Spanish Peseta). Is there really any use for them? I think they just slow down navigation.

**Comment by friedbunny** [comment]
While we're reporting general issues, [Calcbot/Model/Localizable.strings](https://github.com/tapbots/calcbot-localization/blob/12a5f0beddee56368bdad80655354a72d41402f0/ja.lproj/Calcbot/Model/Localizable.strings#L89-90) has `reeverse power` as a key/string pair. I'm assuming this is a typo and not a VoiceOver thing.

**Comment by tapbots** [comment]
First thanks for the comments.

@inikitin Belarusian was indeed spelled incorrectly, I've fixed that. Rouble/Ruble are both "correct" in English. Rouble is I believe the technical dictionary spelling, Ruble is what everyone seems to use. I've changed this to Ruble.

Renminbi/Yuan is a tricky issue, we've tried both alone in the past and have gotten a lot of complaints about the currency missing. In the end, I think its best to have both.

The obsolete currencies are also something that some people use, I think I tried removing them once and got a bunch of complaints. Your point about navigation is well taken though, somehow we forgot to add search to that and its something I plan on getting fixed.

@friedbunny It is a typo, I've fixed it.
